src/fr/invinciblecomics/build.gradle

```
@@ -0,0 +1,8 @@
+ext {
+    extName = 'Invincible Comicsvf'
+    extClass = '.InvincibleComics'
+    extVersionCode = 9
```

Define the initial version as 1
```suggestion
    extVersionCode = 1
```